examples/hmc-agent/hmc_server.container

```diff
@@ -0,0 +1,19 @@
+[Unit]
+Description=Run simple HMC AI agent application and MCP server
+Requires=vllm.service hmc_config.service
+After=vllm.service hmc_config.service
```

Comment:
vllm.service is required only for agent not for server. I can see you have added this also as requirement there.
But its ok to start the server before vllm. You can add vllm.service as one more requirement of agent service